lms/static/scripts/frontend_apps/config.js

```diff
@@ -61,8 +61,10 @@ import { createContext } from 'preact';
  * @prop {Object.<string,string>} formFields
  * @prop {Object} canvas
  *   @prop {boolean} canvas.enabled
+ *   @prop {boolean} canvas.groupsEnabled
```

Comment:
Added this here, couldn't find other references to `FilePickerConfig` but not sure  if needed somewhere else